src/components/SearchBar.js

```diff
@@ -1,14 +1,19 @@
-import React, { Component } from 'react'
-import ResultsList from './containers/ResultsList'
-import search from '../js/search'
```

Comment:
That's what I needed help with. I'm not all too familiar with the JsSearch api. I gave it a few attempts but was unsuccessful in tying the data to the autocomplete form.